components/builder-worker/src/runner/mod.rs

```diff
@@ -300,7 +327,7 @@ impl Runner {
                                   err);
                 streamer.println_stderr(msg)?;
                 self.fail(net::err(ErrCode::POST_PROCESSOR, "wk:run:postprocess"));
-                tx.send(self.job().clone()).map_err(Error::Mpsc)?;
+                tx.send(self.job().clone()).await.unwrap(); // map_err(Error::Mpsc)?;
```

Comment:
Can we use `map_err(Error::Mpsc)?;` here?

Reply:
Addressed in a33367f0febdd930dfb5cba3892af203fc5b3a51